torchvision/extension.py

```diff
@@ -24,6 +24,8 @@ def _register_extensions():
     _register_extensions()
     _HAS_OPS = True
 except (ImportError, OSError):
+    import traceback
+    traceback.print_exc()
```

Comment:
Do we want to keep this or is this debug code?

I'm not saying we should remove this, but this is a different behavior than what we had before